github/projects.go

```diff
@@ -466,3 +466,127 @@ func (s *ProjectsService) MoveProjectCard(ctx context.Context, cardID int64, opt
 
 	return s.client.Do(ctx, req, nil)
 }
+
+type ProjectCollaboratorOptions struct {
+	// Permission specifies the permission to grant to the collaborator.
+	// Possible values are:
+	//     read - can read, but not write to or administer this project.
+	//     write - can read and write, but not administer this project.
+	//     admin - can read, write and administer this project.
+	//
+	// Default value "write"
+	// Note that, if you choose not to pass any parameters,
+	// you'll need to set Content-Length to zero when calling out to this endpoint.
+	Permission string `json:"permission,omitempty"`
+}
+
+// AddProjectCollaborator adds a collaborator to a an organization project and sets
+// their permission level. You must be an organization owner or a project admin to add a collaborator.
+//
+// Github API docs: https://developer.github.com/v3/projects/collaborators/#add-user-as-a-collaborator
+func (s *ProjectsService) AddProjectCollaborator(ctx context.Context, id int64, username string, opt *ProjectCollaboratorOptions) (*Response, error) {
+	u := fmt.Sprintf("projects/%v/collaborators/%v", id, username)
+	req, err := s.client.NewRequest("PUT", u, opt)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeProjectsPreview)
+
+	return s.client.Do(ctx, req, nil)
+}
+
+// RemoveProjectCollaborator removes a collaborator from an organization project.
+// You must be an organization owner or a project admin to remove a collaborator.
+//
+// Github API docs: https://developer.github.com/v3/projects/collaborators/#remove-user-as-a-collaborator
+func (s *ProjectsService) RemoveProjectCollaborator(ctx context.Context, id int64, username string) (*Response, error) {
+	u := fmt.Sprintf("projects/%v/collaborators/%v", id, username)
+	req, err := s.client.NewRequest("DELETE", u, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeProjectsPreview)
+
+	return s.client.Do(ctx, req, nil)
+}
+
+type ListCollaboratorOptions struct {
+	// Affiliation specifies how collaborators should be filtered by their affiliation.
+	// Possible values are:
+	//     outside - All outside collaborators of an organization-owned repository
+	//     direct - All collaborators with permissions to an organization-owned repository,
+	//              regardless of organization membership status
+	//     all - All collaborators the authenticated user can see
+	//
+	// Default value is "all".
+	Affiliation string `url:"affiliation,omitempty"`
```

Comment:
For the same reasons, let's make this a `*string`.

However, I think this needs to be `json` and not `url`:

```go
Affiliation *string `json:"affiliation,omitempty"`
```